release.md

```diff
@@ -18,10 +14,138 @@ pagination:
   <div class="container text-center">
     <div class="row">
       <div class="col-lg-10 mx-auto lead text-left">
-        {%- for post in collections.release reversed -%}
-	        <h3 class="release-heading"><a href="{{ post.url | url }}">{{ post.data.title }}</a></h3>{{ post.date | dateString }}
-	        <p></p>	
-		{%- endfor -%}
+        {%- for pkg in collections.pkgCatalog -%}
+          <article>
+						<h2>{{ pkg[0] }}</h2>
+						{%- assign package = pkg[1] -%}
+						<header>
+							<p>
+							<strong>Latest stable</strong>
+							<a href="{{ package.latestStable.url }}">{{ package.latestStable.version }}</a>
+							</p>
+							<p>
+							<strong>Latest release</strong>
```

Comment:
Totally possible.  It will call for slightly different logic, but that’s simple enough.

The way I have it working now is:
* Find latest stable release and list it
* If that’s also the most recent release, list it in both boxes
   * If not, find the most recent release and list it

If I understand what you’re asking for, it’s:

* Find latest stable release and list it
* Find latest non-stable release and list it, even if older than the stable release

Correct?

Reply:
Yes, that's right!

I think there is not a lot of value in showing the same release as both “stable” and “most recent”, which is why IMO it makes sense to name them “stable” and “development”. But dunno, I am open to consider other options (like of using “latest stable” and “most recent release”).

Reply:
Got it.  Can do!